client/app/routes/show-project.js

```diff
@@ -26,6 +26,9 @@ export default class ShowProjectRoute extends Route {
   @action
   error(e) {
     console.log(e); // eslint-disable-line
-    // this.transitionTo('not-found', 'not-found');
+    const notFoundErrorExists = e.errors.map(err => err.status).includes('404');
```

Comment:
how about 'isNotFound'?

Reply:
also at first testing for the string `404` seems little brittle, but i don't know if there is a better way.